var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -263,3 +267,49 @@ def cmake_args(self):
             self.set_cray_login_node_cmake_options(spec, cmake_args)
 
         return cmake_args
+
+    def setup_environment(self, spack_env, run_env):
+        """Set up the compile and runtime environments for a package."""
+
+        # Environment settings for cbtf-krell
+        run_env.prepend_path('PATH', self.prefix.bin)
+        run_env.prepend_path('PATH', self.prefix.sbin)
+
+        # Find openspeedshop library path
+        cbtfk_libdir = find_libraries('libcbtf-core.so.1.1.0',
+                                      root=self.prefix,
+                                      shared=True, recursive=True)
+        run_env.prepend_path('LD_LIBRARY_PATH',
+                             os.path.dirname(cbtfk_libdir.joined()))
+
+        cbtf_mc = '/sbin/cbtf_mrnet_commnode'
+        cbtf_lmb = '/sbin/cbtf_libcbtf_mrnet_backend'
+        run_env.set('XPLAT_RSH', 'ssh')
+        run_env.set('MRNET_COMM_PATH',
+                    join_path(self.spec.prefix + cbtf_mc))
```

Comment:
You can do `self.prefix.sbin.cbtf_mrnet_commnode`

Reply:
Thanks - changed it.